drivers/hmis/app/graphql/types/hmis_schema/assessment_input.rb

```diff
@@ -16,8 +16,18 @@ class HmisSchema::AssessmentInput < Types::BaseInputObject
     argument :confirmed, Boolean, 'Whether warnings have been confirmed', required: false
     argument :validate_only, Boolean, 'Validate assessment but don\'t submit it', required: false
 
+    # FIXME: this method probably doesn't belong on the form input class
```

Comment:
Yea this should be refactored, it was put here as a quick way for `SaveAssessment` and `SubmitAssessment` to share some setup logic since they have the same input type